h/models/annotation.py

```diff
@@ -12,6 +12,58 @@
 from h.util.user import split_user
 
 
+class AnnotationSlim(Base):
+    __tablename__ = "annotation_slim"
+
+    id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
+    """PK on this table"""
+
+    pubid = sa.Column(types.URLSafeUUID, unique=True)
+
+    """The value of annotation.id, named here pubid following the convetion of group.pubid"""
+
+    created = sa.Column(
+        sa.DateTime,
+        default=datetime.datetime.utcnow,
+        server_default=sa.func.now(),
+        nullable=False,
+        index=True,
+    )
+
+    updated = sa.Column(
+        sa.DateTime,
+        server_default=sa.func.now(),
+        default=datetime.datetime.utcnow,
+        nullable=False,
+        index=True,
+    )
+
+    deleted = sa.Column(
+        sa.Boolean,
+        nullable=False,
+        default=False,
+        server_default=sa.sql.expression.false(),
+    )
+
+    moderated = sa.Column(
```

Comment:
Moderated is now another table, as far as I can tell it doesn't hold any extra data over a boolean ATM.